mako_rollup/src/main.rs

```diff
@@ -18,7 +29,24 @@ fn main() {
 
         // We only care about files, so we intentionally do nothing with directories
         if metadata.file_type().is_file() {
-            let account_uuid = &entry.path().parent().unwrap().to_str().unwrap()[7..];
+            let account_uuid;
+            let mut components = entry.path().components();
+            let next;
+
+            assert_eq!(components.next(), Some(Component::RootDir));
+            assert_eq!(components.next(), Some(Component::Normal(OsStr::new("manta"))));
+
+            next = components.next();
```

Comment:
As you're going to unwrap anyway, this can be

let component = components.next().unwrap();

if component.as_os_str() == "v2" {
  ..
} else {
  account_uuid = component.as_os_str().to_str().unwrap();
}

Reply:
Much better. Thanks!